docs/_data/contributors/gen_contributors.py

```diff
@@ -44,23 +60,50 @@
        **{contributor["name"]}**
 
        ^^^
-        
+
        .. image:: https://github.com/{contributor["handle"]}.png?size=125
 
-       **@{contributor["handle"]}**
- 
+       `@{contributor["handle"]} <https://github.com/{contributor["handle"]}>`_
+
        {contributions}
 
+       {teams}
+
        +++
        {contributor["affiliation"]}
-    """), "   ")
+    """
+        ),
+        "   ",
+    )
+
+# team_titles = {
```

Comment:
If you'd like to do something with these sections later on, could you leave a `# TODO`  comment to explain why they're commented out?

Reply:
Added a note about what this was for - we could have a list like the 'inactive members' list for each team. Or we could list teams on the cards. Either way!